moveit_py/src/moveit/moveit_core/robot_state/robot_state.cpp

```diff
@@ -0,0 +1,507 @@
+           py::arg("joint_model_group_name"),
+           R"(
+           For a given group, get the velocity values of the variables that make up the group.
+
+           Args:
+               joint_model_group_name (str): The name of the joint model group to copy the velocities for.
+           Returns:
+               :py:class:`numpy.ndarray`: The velocities of the joints in the joint model group.
+       )")
+
+      .def("set_joint_group_accelerations",
+           py::overload_cast<const std::string&, const Eigen::VectorXd&>(
+               &moveit::core::RobotState::setJointGroupAccelerations),
+           py::arg("joint_model_group_name"), py::arg("acceleration_values"),
+           R"(
+           Sets the accelerations of the joints in the specified joint model group.
+
+           Args:
+               joint_model_group_name (str): The name of the joint model group to set the accelerations for.
+               acceleration_values (:py:class:`numpy.ndarray`): The accelerations of the joints in the joint model group.
+           )")
+
+      .def("get_joint_group_accelerations", &moveit_py::bind_robot_state::copy_joint_group_accelerations,
+           py::arg("joint_model_group_name"),
+           R"(
+           For a given group, get the acceleration values of the variables that make up the group.
+           Args:
+               joint_model_group_name (str): The name of the joint model group to copy the accelerations for.
+           Returns:
+               :py:class:`numpy.ndarray`: The accelerations of the joints in the joint model group.
+           )")
+
+      // Forward kinematics
+      .def("get_global_link_transform", &moveit_py::bind_robot_state::get_global_link_transform, py::arg("link_name"),
+           R"(
+       Returns the transform of the specified link in the global frame.
+       Args:
+           link_name (str): The name of the link to get the transform for.
+       Returns:
+           :py:class:`numpy.ndarray`: The transform of the specified link in the global frame.
+       )")
+
+      // Setting state from inverse kinematics
+      .def(
+          "set_from_ik",
+          [](moveit::core::RobotState& robot_state, const std::string& group, const geometry_msgs::msg::Pose& pose,
+             const std::string& tip, double timeout) {
+            return robot_state.setFromIK(robot_state.getJointModelGroup(group), pose, tip, timeout);
+          },
+          py::arg("joint_model_group_name"), py::arg("geometry_pose"), py::arg("tip_name"), py::arg("timeout") = 0.0,
+          R"(
+           Sets the state of the robot to the one that results from solving the inverse kinematics for the specified group.
+           Args:
+               joint_model_group_name (str): The name of the joint model group to set the state for.
+               geometry_pose (geometry_msgs.msg.Pose): The pose of the end-effector to solve the inverse kinematics for.
```

Comment:
`py::class`.

Generally, try check the usage of this through the whole PR.

Reply:
[b026ee5](https://github.com/ros-planning/moveit2/pull/1546/commits/b026ee57f3a9af2f9e41c124255b7e577b07b685)